.gitignore

```diff
@@ -13,3 +13,4 @@ doc/manual.xml
 \#*
 *.orig
 regtest-ref
+JSONtest
```

Comment:
The gitignore is not intended for specific tests purpose during development. Please delete this line

Reply:
yes, I had it there only not to synchronise the directory, I am going to make the regtests from the content of the above directory and will remove it from here, thanks! :)

Reply:
is this resolved eszter? if so please press "resolve conversation"